azure/durable_functions/decorators/decorators.py

```diff
@@ -0,0 +1,139 @@
+from .triggers import OrchestrationTrigger, ActivityTrigger, EntityTrigger, EntityClient, OrchestrationClient, DurableClient
+from typing import Callable, Dict, List, Optional
+from azure.durable_functions.entity import Entity
+from azure.durable_functions.orchestrator import Orchestrator
+from typing import Union
+from azure.functions import FunctionRegister, TriggerApi, BindingApi, AuthLevel
+
+class DurableFunctionApp(FunctionRegister, TriggerApi, BindingApi):
+
+    def __init__(self,
+                http_auth_level: Union[AuthLevel, str] = AuthLevel.FUNCTION):
+        """Constructor of :class:`FunctionApp` object.
+        :param http_auth_level: Determines what keys, if any, need to be
+        present
+        on the request in order to invoke the function.
+        """
+        super().__init__(auth_level=http_auth_level)
+
+    def _configure_entity_callable(self, wrap) -> Callable:
+        """Decorator function on user defined function to create and return
+            :class:`FunctionBuilder` object from :class:`Callable` func.
+        """
+
+        def decorator(entity_func):
+            handle = Entity.create(entity_func)
+            handle.__name__ = entity_func.__name__
+
+            return wrap(handle)
+
+        return decorator
+
+    def _configure_orchestrator_callable(self, wrap) -> Callable:
+        """Decorator function on user defined function to create and return
+            :class:`FunctionBuilder` object from :class:`Callable` func.
+        """
+
+        def decorator(orchestrator_func):
+            handle = Orchestrator.create(orchestrator_func)
+            handle.__name__ = orchestrator_func.__name__
+
+            return wrap(handle)
+
+        return decorator
+
+    def on_orchestration_change(self, name: str,
```

Comment:
sorry did not bring this up to you David, pystein decorator naming convention changed, can you change these decorators to follow as below:
for trigger decorator methods, change to <triggertype>_trigger (ex, from on_orchestration_change -> orchestration_trigger)
<img width="590" alt="image" src="https://user-images.githubusercontent.com/12449837/186713673-81d557e0-8cd4-4ca3-b44b-409bd580e077.png">

for input binding methods, change to <inputtype>_input
<img width="535" alt="image" src="https://user-images.githubusercontent.com/12449837/186713566-1e8a35e1-74ce-4615-8ec1-d7191eddd13e.png">

for output binding methods, change to <outputtype>_output
<img width="608" alt="image" src="https://user-images.githubusercontent.com/12449837/186714702-53aa4323-bee3-456f-b087-91724c7a2850.png">